attribute-masking-service/src/contract-tests/resources/application-akka-test.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018-2021 Crown Copyright
```

Comment:
Looks like application-akklatest.yaml and application-dbtest.yaml are still in the repo and should be removed.

Reply:
removed, good spot